src/components/DeviceChecker/DeviceChecker.vue

```diff
@@ -437,5 +438,6 @@ export default {
 
 ::v-deep .modal-container {
 	display: flex !important;
+	width: 750px !important;
```

Comment:
instead of making the container very wide which again shows weird grey space around the video could we show a tooltip with the full name of the device on hover?

Reply:
Regarding this ticket, I think we need to customize from the 3rd library.
https://nextcloud-vue-components.netlify.app/#/Components/Multiselect